python/cudf/cudf/_lib/reduce.pyx

```diff
@@ -105,3 +105,24 @@ def scan(scan_op, Column incol, inclusive, **kwargs):
 
     py_result = Column.from_unique_ptr(move(c_result))
     return py_result
+
+
+def minmax(Column incol):
+    """
+    Top level Cython minmax function wrapping libcudf++ minmax.
+
+    Parameters
+    ----------
+    incol : Column
+        A cuDF Column object
```

Comment:
Needs a `Returns` section.

Side note/opinion: these are internal functions so we don't really need to document them in full numpydoc style. Just a one-liner explaining what the function returns is sufficient. Documenting internals in so much detail can lead to a lot of unnecessary churn.